### PR TITLE
Add option to add custom url to workspace

### PR DESCRIPTION
**Pull request by superdav42**
# Describe what your pull request addresses

- [ ] #8 
- [ ] This just adds the option to the UI it doesn't actually work yet. 

image

### CODE REVIEW

**Review by mandolyte — APPROVED** [review]

**Comment by mandolyte** [comment]
Marked as a draft PR (work in progress); looks good so far. Do you want to merge to main?

If so, remove the draft status, let me know, and I'll finish it up.
@superdav42

**Comment by superdav42** [comment]
> Second there is a remnant of a URL input field in the app bar. Left by mistake?

Yes, I just removed it.

I haven't made it actually fetch the content yet. I was going to wait until Lars PR is merged since it'll probably change many things related to that. This is why it's in draft.

**Comment by superdav42** [comment]
This is ready for review now. Content is loaded from the provided URL.

**Pull request merged**
